atomix/cluster/src/main/java/io/atomix/cluster/discovery/DnsDiscoveryBuilder.java

```diff
@@ -16,34 +16,10 @@
  */
 package io.atomix.cluster.discovery;
 
-import java.time.Duration;
-
 /** DNS discovery builder. */
 public class DnsDiscoveryBuilder extends NodeDiscoveryBuilder {
```

Comment:
@npepinpe I assume we don't want to use the DNS discovery in the near future? So maybe we can remove it completely?